backend/src/main/java/ch/sectioninformatique/template/item/Item.java

```diff
@@ -1,92 +0,0 @@
-package ch.sectioninformatique.template.item;
```

Comment:
Why to Delete all the item folder's content in this project ? This is used to provide an example package for future users of the template_frontback project.